### PR TITLE
Bug:1184837 - [mobile] Inconsistent button size

### PR DESCRIPTION
**Pull request by arush0311**

### CODE REVIEW

File: src/media/css/buttons-loadmore.styl [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /* 'more' button container, button, placement, and spinners. */
 @import 'lib';
 
+
```
Comment (atiqueahmedziad) [inline src/media/css/buttons-loadmore.styl]: Please, delete this extra line you added

File: src/media/css/buttons-loadmore.styl [hunk 2]
```diff
@@ -79,3 +80,9 @@ $loadmore-desktop-width = 300px;
         padding-bottom: 0;
     }
 }
+
+@media $base-tablet{
```
Comment (atiqueahmedziad) [inline src/media/css/buttons-loadmore.styl]: give a space after $base-tablet 
`+@media $base-tablet {`

**Comment by arush0311** [comment]
Done

**Comment by atiqueahmedziad** [comment]
Please, squash the 3 commits into one,  
`git rebase-i HEAD^3`
pick the 1st one & squash the other two. 
then give cmd
`git push-f`

**Comment by arush0311** [comment]
Done that

**Comment by atiqueahmedziad** [comment]
r?

**Comment by arush0311** [comment]
Please explain.Is it fixed?

**Comment by atiqueahmedziad** [comment]
r ? @mstriemer @chuckharmston

**Comment by ngokevin** [comment]
looks good to me r+ thanks!

**Pull request merged**